src/python/dxpy/nextflow/nextflow_utils.py

```diff
@@ -50,7 +50,7 @@ def get_regional_options(region):
         region: {
```

Comment:
Minor: maybe it is better looking this way:
```
    nextflow_asset, nextaur_asset = get_nextflow_assets(region)
    regional_instance_type = get_instance_type(region)
    
    regional_options = {
        region: {
            "systemRequirements": {
                "*": {
                    "instanceType": regional_instance_type
                }
            },
            "assetDepends": [
                {"id": nextaur_asset},
                {"id": nextflow_asset}
            ]
        }
    }

Reply:
Actually since systemRequirements is optional, I do wonder if we need it at build time. Because the user might not realize that it is specified here, and they actually want the default to be specified based on for example their own project/billto settings. cc @yihchii @commandlinegirl

Reply:
We have defaults in all of our apps in file-apps - but I agree that it might be different in the future. 

I recommend we merge this, but discuss this next week with Ola.